addons/l10n_es_edi_tbai/models/web_services.py

```diff
@@ -14,248 +14,239 @@
 from odoo.tools import get_lang
 from odoo.tools.xml_utils import cleanup_xml_node
 
-# -------------------------------------------------------------------------
-# TICKETBAI WEB SERVICES
-# -------------------------------------------------------------------------
+
+# ===== TicketBAI WEB SERVICES =====
+# Provides helper methods for interacting with the Basque country's TicketBai servers.
```

Comment:
why not put it in account_edi_format.py instead?

Reply:
No particular reason other than it's pretty long. Let me know what you prefer.